src/DateTimePickerAndroid.android.ts

```diff
@@ -128,6 +131,7 @@ function open(props: AndroidNativeProps) {
 
 function dismiss(mode: AndroidNativeProps['mode']): Promise<boolean> {
   // $FlowFixMe - `AbstractComponent` [1] is not an instance type.
+  // @ts-ignore
```

Comment:
if you import from `pickers.android`, there won't be need for this `  // @ts-ignore`